utils/src/ooinstall/cli_installer.py

```diff
@@ -261,23 +307,32 @@ def check_hosts_config(oo_cfg):
             if master_lb[0].master or master_lb[0].node:
                 click.echo('The Master load balancer is configured as a master or node. Please correct this.')
                 sys.exit(0)
+            # Check for another host with same connect_to?
         else:
             message = """
-No HAProxy given in config. Either specify one or provide a load balancing solution
-of your choice to balance the master API (port 8443) on all master hosts.
+WARNING: No master load balancer specified in config. If you proceed you will
+need to provide a load balancing solution of your choice to balance the
+API (port 8443) on all master hosts.
```

Comment:
Well, technically, they could specify the load-balancer name under openshift_master_named_certificates for the cert part, but we would need to provide a way to specify what we use for the generated kubeconfigs...

It gets even more complicated in cloud based installs, because ideally they would have an internal and an external load balancer...  one to load balance console/api traffic for end users and one for load balancing internal traffic that doesn't traverse the external network.